config/default.yml

```diff
@@ -1172,10 +1172,13 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   CountComments: false  # count full line comments?
   Max: 25
-  Exclude:
-    - 'Rakefile'
-    - '**/*.rake'
-    - 'spec/**/*.rb'
+  ExcludedMethods:
```

Comment:
Okay, that's what I did now:

* no exclusions (of any kind) in the default config
* rubocop uses the old (file based) exclusion for its own validation